msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/crosscloud/TestCase1592465.java

```diff
@@ -117,7 +122,17 @@ public void test_acquire_token_from_cross_cloud_after_acquiring_token_from_home_
         Assert.assertEquals("Verify Exception operation name", "authority", exception.getOperationName());
 
         // Acquire token interactively from cross cloud, expected to get a new access token
-        final OnInteractionRequired crossCloudInteractionHandler = () -> {};
+        final OnInteractionRequired crossCloudInteractionHandler = () -> {
+            final PromptHandlerParameters promptHandlerParameters =
+                    PromptHandlerParameters.builder()
+                            .prompt(PromptParameter.SELECT_ACCOUNT)
+                            .loginHint(userName)
+                            .broker(mBroker)
+                            .staySignedInPageExpected(GlobalConstants.IS_STAY_SIGN_IN_PAGE_EXPECTED)
+                            .build();
+            final AadPromptHandler promptHandler = new AadPromptHandler(promptHandlerParameters);
+            promptHandler.handlePrompt(userName, password);
```

Comment:
So this was previously passing without a prompt handler I guess.

Reply:
I think the Expected Results in the Test Case in ADO are not clear. @iamgusain Can you review that test case in ADO ensure that Expected Results column clearly indicates whether to expect a prompt or not?

Reply:
This case and `1592510` are behaving differently on pipeline, the second request is not prompting for credentials. On my physical device, the second request always prompts for credentials. I'm probably just going to ignore these two for now and look at them later.